mrp_operations_extension/__openerp__.py

```diff
@@ -6,7 +6,7 @@
 
 {
     "name": "Manufacturing Operations Extension",
-    "version": "8.0.2.0.0",
+    "version": "8.0.3.0.0",
```

Comment:
In this case it should be 8.0.2.1.0, as is a non-breaking feature but requires to update the module.